pyvrp/cli.py

```diff
@@ -267,6 +272,14 @@ def main():
     """
     parser.add_argument("--config_loc", help=msg)
 
+    msg = "Node operators to apply. Defaults to all."
+    node_ops = [op.__name__ for op in NODE_OPERATORS]
+    parser.add_argument("--node_ops", nargs="+", default=node_ops, help=msg)
+
+    msg = "Route operators to apply. Defaults to all."
+    route_ops = [op.__name__ for op in ROUTE_OPERATORS]
+    parser.add_argument("--route_ops", nargs="+", default=route_ops, help=msg)
```

Comment:
How about we move the ops to the config? That to me feels like the right place to define the algorithm configuration, which LS operators are part of.